src/objective-c/tests/run_one_test_bazel.sh

```diff
@@ -48,4 +48,7 @@ $INTEROP --port=$TLS_PORT --max_send_message_size=8388608 --use_tls &
 
 trap 'kill -9 `jobs -p` ; echo "EXIT TIME:  $(date)"' EXIT
 
-../../../tools/bazel run $SCHEME
+time \
```

Comment:
this change should actually be in grpc_objc_bazel_test.sh as well, because the run_one_test_bazel.sh will probably be removed at some point (once all the tests migrate to grpc_objc_baze_test.sh)

Reply:
https://github.com/grpc/grpc/blob/9c23d7999ce827614195fe2da6653af30cb4298c/tools/internal_ci/macos/grpc_objc_bazel_test.sh#L87  

which currently run MacTest/UnitTest and codegen plugin test

Reply:
chatted offline, will update grpc_objc_bazel_test with env var as well